x/gov/wire.go

```diff
@@ -0,0 +1,13 @@
+package gov
+
+import (
+	"github.com/cosmos/cosmos-sdk/wire"
+)
+
+// TODO: Delete?
```

Comment:
nope this should stay in here it should be used in the app.go when constructing the codec.

Reply:
(it's currently used) - so yeah I'd remove this comment